src/modules/TextOverlay/info.json

```diff
@@ -8,6 +8,10 @@
             "desc": "Enter the text to overlay.",
             "default": "Lorem ipsum"
         },
+        "Custom coordinates": {
+            "type": "button",
```

Comment:
Or maybe we can just make a new type instead of making a button and then giving it an id. I feel that will be better.

Reply:
which type we should give then?